newrelic/hooks/framework_django.py

```diff
@@ -600,10 +606,9 @@ def _wrapped(path):
             with FunctionTrace(name=name, label=path, source=wrapped):
                 result = wrapped(path)
 
-                if type(result) is tuple:
+                if isinstance(result, tuple):
```

Comment:
Sometimes there are good reasons for using the type() is instead of isinstance check, I suspect this is likely one of those cases so we should probably change this back to type(result) is tuple. Sometimes an isinstance check will return true while the type is check will return false (in inheritance cases). I wouldn't risk breaking the logic here which is why I say, while I don't know for sure without a deeper investigation, it's safer to leave it as it was.
```suggestion
                if type(result) is tuple:
```